client/a8c-for-agencies/components/pressable-usage-details/style.scss

```diff
@@ -86,4 +86,91 @@
 		@include heading-large;
 		margin-top: 5px;
 	}
+
+	/* Titan Email Addon styles */
+	.pressable-usage-details__addon-tag {
+		@include body-small;
+		background-color: var(--color-neutral-5);
+		color: var(--color-neutral-70);
+        font-size: 11px;
+		padding: 2px 6px;
+		border-radius: 4px;
+		margin-left: 6px;
```

Comment:
Nitpicking: For margins and padding. Can we use logical properties instead?

e.g.
```
margin-inline-start: 6px;
```